### PR TITLE
refactor(UNT-T11378): code audit changes

### PR DESCRIPTION
**Pull request by het-a-simform**
- update component level readme file.
- update prop description
- import types in index file.

### CODE REVIEW

File: src/components/AnimatedScanner/README.md [hunk 1]
```diff
@@ -71,9 +69,9 @@ export default AnimatedScanner;
 | height           | 200                    | number  | `height` to set `AnimatedScanner` component height.             |
 | style            | `{{ marginTop: 100 }}` | object  | `style` to set component style.                                 |
 | zoomingDelay     | 800                    | number  | `zoomingDelay` use for outer component zoomingDelay.            |
-| bordercolor      | ColorSet.Primary       | string  | `borderColor` use to give color for Outer zooming component.    |
+| bordercolor      | #01579B                | string  | `borderColor` use to give color for Outer zooming component.    |
```
Comment (shalin-simform) [inline src/components/AnimatedScanner/README.md:72]: make o smaller

File: src/components/AnimatedSwitch/hooks/useAnimatedSwitch.ts [hunk 2]
```diff
@@ -11,8 +11,10 @@ const useAnimatedSwitch = ({
   size,
   lightThemeColor,
   darkThemeColor,
+  onToggle,
 }: AnimatedSwitchProps) => {
   const [darkTheme, setDarkTheme] = useState(false);
+  const [statusofSwitch, setStatusOfSwitch] = useState('light');
```
Comment (shalin-simform) [inline src/components/AnimatedSwitch/hooks/useAnimatedSwitch.ts:17]: remove this if not needed

File: src/components/AnimatedSwitch/hooks/useDayAndNightAnimatedSwitch.ts [hunk 3]
```diff
@@ -13,6 +13,7 @@ const useDayAndNightAnimatedSwitch = ({
   size,
 }: DayAndNightAnimatedSwitchProps) => {
   const [darkTheme, setDarkTheme] = useState(false);
+  const [statusofSwitch, setStatusOfSwitch] = useState(false);
```
Comment (shalin-simform) [inline src/components/AnimatedSwitch/hooks/useDayAndNightAnimatedSwitch.ts:16]: remove this if not needed

File: src/components/AnimatedSwitch/README.md [hunk 4]
```diff
@@ -67,11 +65,12 @@ export default AnimatedSwitch;
...
-| :-------------- | :-------- | :----- | :------------------------------ |
-| size            | `100`     | number | `size` of `switch`.             |
-| lightThemeColor | `#59abdd` | string | `switch color` when it's `OFF`. |
-| darkThemeColor  | `#0f1341` | string | `switch color` when it's `ON`   |
+| Props           | Default   | Type    | Description                                                   |
+| :-------------- | :-------- | :------ | :------------------------------------------------------------ |
+| size            | `100`     | number  | `size` of `switch`.                                           |
+| lightThemeColor | `#59abdd` | string  | `switch color` when it's `OFF`.                               |
+| darkThemeColor  | `#0f1341` | string  | `switch color` when it's `ON`                                 |
+| onToggle        | `false`   | boolean | `onToggle` user get the value of swith like ex. light or dark |
```
Comment (shalin-simform) [inline src/components/AnimatedSwitch/README.md:73]: update default value

File: src/components/AnimatedCheckMark/hooks/useCircularAnimatedCheckMark.ts [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import { useEffect, useRef } from 'react';
 import { Animated } from 'react-native';
-import type { CircularAnimatedStyle } from '../type';
+import type { CircularAnimatedStyle } from '../types';
```
Comment (mukesh-simform) [inline src/components/AnimatedCheckMark/hooks/useCircularAnimatedCheckMark.ts:3]: @het-a-simform It should be CircularAnimatedProps not Style

File: src/components/AnimatedCheckMark/hooks/useSimpleAnimatedCheckMark.ts [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import { useEffect, useRef } from 'react';
 import { Animated } from 'react-native';
-import type { AnimationStyleProps } from '../type';
+import type { AnimationStyleProps } from '../types';
```
Comment (mukesh-simform) [inline src/components/AnimatedCheckMark/hooks/useSimpleAnimatedCheckMark.ts:3]: @het-a-simform It's not a style prop so you should rename this

**Review by mukesh-simform — APPROVED** [review]

**Pull request merged**